web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -588,7 +591,7 @@
             <VideoViewer
               assetId={previewStackedAsset.id}
               on:close={closeViewer}
-              on:onVideoEnded={handleVideoEnded}
+              on:onVideoEnded={() => navigateAsset()}
```

Comment:
```suggestion
              on:onVideoEnded={navigateAsset}
```

Reply:
No you can't because `e: CustomEvent<void>` != `order?: 'previous' | 'next'`